nntrainer/tensor/manager.cpp

```diff
@@ -498,7 +508,7 @@ Manager::requestInputs(const GraphNode &node,
   using RT = TensorSpecV2::RequestType;
 
   TensorSpecV2 var_common_spec, grad_common_spec;
-  var_common_spec.ls = TensorLifespan::FORWARD_GRAD_LIFESPAN;
+  var_common_spec.ls = TensorLifespan::FORWARD_GRAD_AGRAD_LIFESPAN;
```

Comment:
I think it should be FORWARD_GRAD_LIFESPAN. We do not need the Input Tensor for ```Applying Gradient```. Could you check plz?

Reply:
It seems AGRAD is unnecessary. It passed test without it.
thianks.